rules/backend.yar

```diff
@@ -245,10 +245,8 @@ rule md5_f797dd5d8e13fe5c8898dbe3beb3cc5b {
 }
 rule md5_22927e670b0953c01874d626110aa474 {
     strings:
-        $checkout_page_regex = "new RegExp(\"onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout"
-        $change_detection = ".on(\"change"
-        $payment_field = "input[name=\"payment"
-    condition: all of them
+        $ =  /(RegExp[^\)]*(onepage|nexwaycheckout|checkout|onestep|firecheckout|onestepcheckout)[^\)].+(grelos_v=null))/
```

Comment:
Hi!

- `RegExp[^\)]*` will scan until the end of the document for every instance, if no "onepage" etc is encountered. So you could limit the number of characters that will possibly be matched, like this: `RegExp[^\)]{,3}` 
- I've seen this pattern without grelos_v instances, so I think we can do without it (Magereport already does it, have only seen 2 FPs in the last year).
- Checking for `nexwaycheckout` and `onestepcheckout` and `firecheckout` is redundant if you already check for `checkout`